pr_line_related_po_line/__manifest__.py

```diff
@@ -2,9 +2,9 @@
     "name": "PR Line related PO Line",
     "version": "14.0.1.0.0",
     "author": "Vauxoo",
-    'category': 'Inventory/Purchase',
+    "category": "Inventory/Purchase",
     "website": "http://www.vauxoo.com/",
-    "license": "LGPL-3",
+    "license": "OPL-1",
```

Comment:
License is correct in this case, because modules in public repos like this one or the OCA are expected to be free software